src/api/handlers/media_library.py

```diff
@@ -21,6 +21,67 @@ def registerRoutes(self):
         self.add("/gallery.find", self.find_images)
         self.add("/gallery.item.edit", self.edit_details)
 
+        self.add("/gallery.generate.hashes", self.generate_hashes) # A temporary route that we will need to run to generate hashes of already uploaded content (ONCE!)
+        self.add("/gallery.duplicates.summarize", self.summarize_duplicates) # Generates a CSV of duplicate images with other useful attributes
+        self.add("/gallery.duplicates.clean", self.clean_duplicates) # Allows you to clean all/some duplicates and transfer relationships to only one record
+        self.add("/gallery.duplicates.summary.print", self.print_duplicates) # Allows you to clean all/some duplicates and transfer relationships to only one record
+
+    # @admins_only
+    def print_duplicates(self, request):
```

Comment:
can we make this print summary also include the non-duplicated media?

Reply:
Noted